src/installLogsPrinter.js

```diff
@@ -60,9 +60,10 @@ function installLogsPrinter(on, options = {}) {
     [CONSTANTS.TASK_NAME]: function (data) {
       let messages = data.messages;
 
-      if (typeof options.compactLogs === 'number' && options.compactLogs >= 0) {
-        messages = compactLogs(messages, options.compactLogs);
-      }
+      const terminalMessages =
+        typeof options.compactLogs === 'number' && options.compactLogs >= 0
```

Comment:
In order to not introduce a breaking change let's default to main `compactLogs` option here if this is undefined.